package.json

```diff
@@ -20,6 +20,7 @@
         "mocha": "3.5.3",
         "path": "0.12.7",
         "recursive-readdir": "2.2.1",
+        "sleep": "5.1.1",
```

Comment:
This library blocks the main event loop so shouldn't be used.  Instead, I recommend a little function like:
```
async function sleep(milliseconds: number): Promise<void> {
  return new Promise((resolve, reject) => {
    setTimeout(resolve, milliseconds);
  }
}
```
Then you can use it like:
```
await sleep(500);
```